modules/python/tests/test_python_persist_name.c

```diff
@@ -205,3 +205,51 @@ Test(python_persist_name, test_python_exception_in_generate_persist_name)
   Py_DECREF(persist_generator_persist);
   log_pipe_unref((LogPipe *)p);
 }
```

Comment:
(Comment is for the whole commit)
IMHO we should have one more testcase, which shows the precedence of the `persist_name` config option, and the `generate_persist_name` function. Which one is preferred if both of them given? Should we get a config error?

Reply:
Added unit test for it. The persist name from config takes precedence. No error is generated if both provided.